src/app/components/appointments/appointments.component.ts

```diff
@@ -0,0 +1,28 @@
+import { Component, Input } from '@angular/core';
+import { Charminglook, Treatments } from '../../interfaces/charminglook';
+import { FormBuilder, FormGroup } from '@angular/forms';
+import { CommonModule } from '@angular/common';
+
+@Component({
+  selector: 'app-appointments',
+  standalone: true,
+  imports: [CommonModule],
+  templateUrl: './appointments.component.html',
+  styleUrl: './appointments.component.scss',
+})
+export class AppointmentsComponent {
+  constructor(private formBuilder: FormBuilder) {}
+  @Input() title!: Charminglook;
+  @Input() appointments!: Treatments;
```

Comment:
typ, który przychodzi z data.treatments to Treatments[], więc bez arraya w typie będzie sypać błędem